ui.cpp

```diff
@@ -37,6 +51,14 @@ void Ui::update(unsigned long elapsed)
 
     m_display.clear();
 
+    if (m_refresh_indicator) {
+        m_display.draw_pixel(85, 30);
+        m_refresh_indicator = false;
+    } else {
+        m_display.draw_pixel(85, 34);
+        m_refresh_indicator = true;
+    }
```

Comment:
Please drop that.

Reply:
Fixed in 03d51eeef524e32c61277c2647ff848bb0dc321d